src/jit/objectalloc.cpp

```diff
@@ -579,14 +579,12 @@ bool ObjectAllocator::CanLclVarEscapeViaParentStack(ArrayStack<GenTree*>* parent
 
                 if (asCall->gtCallType == CT_HELPER)
                 {
-                    const CorInfoHelpFunc helperNum = comp->eeGetHelperNum(asCall->gtCallMethHnd);
+                    // TODO-ObjectStackAllocation: Special-case helpers here that
```

Comment:
@erozenfeld - is there a tracking issue for this `TODO`?

Reply:
@CarolEidt It's one of the items in todo list in https://github.com/dotnet/coreclr/issues/20253#issue-366568135 :
- [ ] special case calls to helpers where arguments don't escape (this will require ensuring that the helpers report gc arguments as interior to gc)